java-client/src/main/java/co/elastic/clients/json/JsonpMapper.java

```diff
@@ -51,6 +54,11 @@
      */
     <T> void serialize(T value, JsonGenerator generator);
 
+    /**
+     * Serialize an object to its Json String representation
```

Comment:
Nit: capitalize JSON

```suggestion
     * Serialize an object to its JSON String representation
```